lib/rspec/matchers/built_in/has.rb

```diff
@@ -90,8 +90,12 @@ def predicate_method_name
           predicate
         end
 
-        def predicate_matches?
-          !!predicate_result
+        def predicate_matches?(value=true)
```

Comment:
Nitpick: it's strange that RuboCop doesn't catch that, or maybe it's our rules. Do you mind to add some space around `=`?